### PR TITLE
Format Data & Sdcard Mount Fix

### PR DESCRIPTION
**Pull request by SIDDK24**
Fix Format data & empty /sdcard

### CODE REVIEW

File: partition.cpp [hunk 1]
```diff
@@ -1237,7 +1244,8 @@ void TWPartition::Setup_Data_Media() {
 			Make_Dir("/sdcard", false);
 			Symlink_Mount_Point = "/sdcard";
 		}
-		if (Mount(false) && TWFunc::Path_Exists(Mount_Point + "/media/0")) {
```
Comment (AndroiableDroid) [inline partition.cpp:1240]: Here I am having doubt

**Review by AndroiableDroid — COMMENTED** [review]
@imranpopz @RazaDroid check it plz

File: partition.cpp (hunk 1 above)
Comment (SIDDK24) [reply partition.cpp:1240]: https://gerrit.twrp.me/c/android_bootable_recovery/+/5041/22/partition.cpp#1204

Missed this line

Comment (AndroiableDroid) [reply partition.cpp:1240]: Working fine for all devices?

Comment (SIDDK24) [reply partition.cpp:1240]: > Working fine for all devices?

Tested on RMX1941 format data & internal shows after format but format data is still broken on OnePlus 6& 6t

Comment (AndroiableDroid) [reply partition.cpp:1240]: one more last thing, is it missing authorship?

**Pull request closed without merging**